src/pages/search/[id].vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <Head>
+    <Title>{{ APP_TITLE_SHORT }} | Search Results</Title>
+  </Head>
+
+  <page-header>
+    Search Results
+  </page-header>
+
+  <template v-if="!isLoading">
+    <app-tabs>
+      <app-tab title="Names">
+        <search-names-panel class="search__panel"/>
+      </app-tab>
+      <app-tab title="Tokens">
+        <search-tokens-panel class="search__panel"/>
+      </app-tab>
+    </app-tabs>
+
+    <app-panel class="search__panel">
+      <template #heading>
+        ACCOUNTS
+        <hint-tooltip>
+          {{ accountHints.account }}
+        </hint-tooltip>
+      </template>
+      <p class="search__paragraph">
+        No Accounts were found with the string <span class="search__highlighted">ak_-*{{ query }}</span>.
+      </p>
+      <p class="search__paragraph">
+        Please search using the complete <span class="search__highlighted">Account ID</span> to go directly to the
+        result.
+      </p>
+      <p class="search__paragraph">
+        Here are specific instructions as how to find it.
+      </p>
+    </app-panel>
+
+    <app-panel class="search__panel">
+      <template #heading>
+        TRANSACTIONS
+        <hint-tooltip>
+          {{ transactionsHints.transaction }}
+        </hint-tooltip>
+      </template>
+      <p class="search__paragraph">
+        No Transactions were found with the string <span class="search__highlighted">th_-*{{ query }}</span>.
+      </p>
+      <p class="search__paragraph">
+        Please search using the complete <span class="search__highlighted">Transaction Hash</span> to go directly to the
+        result.
+      </p>
+      <p class="search__paragraph">
+        Here are specific instructions as how to find it.
+      </p>
+    </app-panel>
+
+    <app-panel class="search__panel">
+      <template #heading>
+        SMART CONTRACTS
+        <hint-tooltip>
+          {{ contractsHints.contract }}
+        </hint-tooltip>
+      </template>
+      <p class="search__paragraph">
+        No Smart Contracts were found with the string <span class="search__highlighted">th_-*{{ query }}</span>.
```

Comment:
I would prefer to apply `q` styling just like in the error page we have discussed recently in error handling PR code review

Reply:
I dunno, it still does not feel right. If it would be surrounded with quotes `""` and it had italic style I would not hesitate to use the `q`. The solution is at least controversial. Can we come up with some other ideas?